modules/virt-configuring-vm-real-time.adoc

```diff
@@ -49,56 +49,127 @@ spec:
         memory:
           guest: 4Gi
           hugepages:
-            pageSize: 1Gi <6>
+            pageSize: 1Gi # <6>
         terminationGracePeriodSeconds: 0
 # ...
```

Comment:
As discussed - we need a more explicit VM manifest example, that perfectly suits realtime. It must include the `image` entry, because a realtime VM is not a standard one, and the user might not have created and pushed their own image.

Reply:
@phoracek @sjhala-ccs WDYT?

Reply:
BTW, the example DPDK VM spec does not have a disk as well https://docs.openshift.com/container-platform/4.15/virt/vm_networking/virt-using-dpdk-with-sriov.html#virt-configuring-vm-dpdk_virt-using-dpdk-with-sriov

Reply:
This is not the only place where we leave out part of the VM yaml. I think it makes sense to document only the "special" parts, to make this easier to maintaine by making if future-proof, and also put the spotlight on the important attributes.

That being said, I will not fight against having a full example either.